sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/ScalaReflection.scala

```diff
@@ -595,6 +559,13 @@ object ScalaReflection extends ScalaReflection {
       }
     }
   }
+
+  def getConstructorParas(cls: Class[_]): Seq[(String, Type)] = {
```

Comment:
`getConstructorParameters` and can you give some scaladoc / limitations (i.e. assumes a default construct, probably only works for scala classes).